flake.nix

```diff
@@ -0,0 +1,115 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+  };
+
+  outputs = inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      # TODO(XXX): in theory this flake could support aarch64-linux,
+      #   x86_64-darwin and aarch64-darwin, but it is untested.
+      systems = [ "x86_64-linux" ];
+      perSystem = { config, self', pkgs, lib, system, ... }:
+        let
+          buildToolsVersion = "30.0.3";
+          platformVersion = "33";
+          rustTargets = [ "x86_64-linux-android" ];
+
+          devDeps = with pkgs; [
+            android-sdk
+            android-studio
+            maven
+            jdk11 # Matched to CI setup-java task's java-version
+          ];
+
+          android-comp = pkgs.androidenv.composeAndroidPackages {
+            buildToolsVersions = [ buildToolsVersion ];
+            platformVersions = [ platformVersion ];
+            abiVersions = [ "x86_64" ];
+
+            # Matched to CI's Run Android Tests task - we want google_apis to
+            # ensure network access for real-world test suite.
+            systemImageTypes = [ "google_apis" ];
```

Comment:
Leaving myself a note to rm this after https://github.com/rustls/rustls-platform-verifier/pull/74 lands

Reply:
Fixed and pushed, but it seems like there's some lag so the commit hasn't shown up in the UI yet. I'm logging out for the evening but will follow-up tomorrow.